src/main.rs

```diff
@@ -60,15 +64,25 @@ async fn init_maker(
 
 #[tokio::main]
 async fn main() {
-    // TODO: from config or hardcoded
-    let dai_contract_addr: comit::ethereum::Address = unimplemented!();
+    let options = options::Options::from_args();
+
+    let settings = read_config(&options)
+        .and_then(Settings::from_config_file_and_defaults)
+        .expect("Could not initialize configuration");
+
+    let dai_contract_addr: comit::ethereum::Address = settings.ethereum.dai_contract_address;
 
     // TODO: Proper wallet initialisation from config
-    let bitcoin_wallet =
-        bitcoin_wallet::Wallet::new(unimplemented!(), unimplemented!(), unimplemented!()).unwrap();
-    let ethereum_wallet = ethereum_wallet::Wallet::new(unimplemented!(), unimplemented!()).unwrap();
+    let bitcoin_wallet = bitcoin_wallet::Wallet::new(
+        unimplemented!(),
```

Comment:
❓  Where does the seed come from, do we just use `seed.rs` here?

Reply:
Read the description :) The seed is not done.

Reply:
Sorry, missed that one! :)